typespec-extension/src/type-utils.ts

```diff
@@ -263,6 +263,10 @@ export function getAccess(type: Model | Operation | Enum): string | undefined {
   });
 }
 
+export function isAllValueInteger(values: number[]): boolean {
+  return values.every((it) => Number.isInteger(it));
+}
```

Comment:
Not a good way to decide the Enum is integer or float. But typespec does not tell the type.